rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/activity/CovidJsonFormActivity.java

```diff
@@ -104,6 +105,7 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
We can remove this overridden method if it's the same as https://github.com/onaio/rdt-standard/pull/953/files#diff-f8f8bfc77f90eeb4ecb4cef36ec3b6020c965fc37320b8e4e63b5d3934933751R72.

Reply:
actually, I think we cannot remove the overridden method because the child class method checking the location permission whereas the parent class method checking the phone permission.

Reply:
cool